lib/api/models/request.js

```diff
@@ -21,7 +19,7 @@ export class Request {
   buildMockedRequestData() {
 
     const defaultState = {
-      name: 'Original Response',
+      name: 'Mocked Response',
```

Comment:
Ok so lets name it Mocked original response for now, until we have a better idea.

In general i'm working on making the UI easier to understand because It is a bit confusing to use right now.